nucypher/blockchain/eth/actors.py

```diff
@@ -318,9 +302,10 @@ def connect_condition_providers(
                 duplicated_endpoint_check[chain_id].add(uri)
 
         # Ingest default/fallback RPC providers for each chain
-        for chain_id in self.domain.condition_chain_ids:
-            default_endpoints = get_healthy_default_rpc_endpoints(chain_id)
-            for uri in default_endpoints:
+        # for chain_id in self.domain.condition_chain_ids:
+        default_endpoints = get_healthy_default_rpc_endpoints(self.domain.name)
```

Comment:
Should this be `self.domain` now?
```suggestion
        default_endpoints = get_healthy_default_rpc_endpoints(self.domain)
```

Reply:
done